src/transformers/models/vit/configuration_vit.py

```diff
@@ -119,3 +125,16 @@ def __init__(
         self.num_channels = num_channels
         self.qkv_bias = qkv_bias
         self.encoder_stride = encoder_stride
+
+
+class ViTOnnxConfig(OnnxConfig):
+
+    TORCH_ONNX_MINIMUM_VERSION = version.parse("1.12")
```

Comment:
This is where I override the default minimum `torch` version to be the one required for this model